main.py

```diff
@@ -0,0 +1,82 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+# Use a breakpoint in the code line below to debug your script.
+
+
+# Press the green button in the gutter to run the script.
+
+
+# See PyCharm help at https://www.jetbrains.com/help/pycharm/
+def main():
+    first_board = [[], [], [], [], [], [], [], [], [], []]
+    for g in range(0, 10):
+        for k in range(0, 10):
+            first_board[g].append(0)
+            print(first_board[g][k], end=' ')
+        print()
+    z = (input())
+    first_board[3][3] = 1
+    first_board[3][4] = 1
+    first_board[3][5] = 1
+
+    first_board = dead_or_life_condition(first_board)
+    z = (input())
+    first_board = dead_or_life_condition(first_board)
+    z = (input())
+    first_board = dead_or_life_condition(first_board)
+    z = (input())
+    first_board = dead_or_life_condition(first_board)
+    z = (input())
+    first_board = dead_or_life_condition(first_board)
+    z = (input())
+    first_board = dead_or_life_condition(first_board)
+
+
+
+def dead_or_life_condition(board):
+    new_board = [[], [], [], [], [], [], [], [], [], []]
```

Comment:
I'm not entirely sure what those lines (41-44) do. They should be enclosed in a function with an indicative name